tests/src/io/openliberty/tools/eclipse/test/it/LibertyPluginSWTBotMavenTest.java

```diff
@@ -802,4 +723,56 @@ public void testDefaultJRECompliance() {
             SWTBotPluginOperations.closeDialog(bot);
         }
     }
+
+    /**
+     * Tests that a non-Liberty project can be manually be categorized to be Liberty project. This test also tests the refresh
+     * function.
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testAddingProjectToDashboardManually() throws Exception {
+
+        IProject iProject = LibertyPluginTestUtils.getProject(NON_DFLT_NAME);
+        String projectName = iProject.getName();
+
+        // Refresh the project through the explorer view to pick up the nature removal.
```

Comment:
This comment no longer applies. Also the refreshes on line 740 and 743 are a bit redundant because all you really need is the check on line 746-755.

Reply:
OK.  I wonder if I need a refresh of the DB after refreshing the project explorer.  I don't think so but am running the tests now.